crates/ironrdp-web/Cargo.toml

```diff
@@ -42,12 +42,11 @@ iron-remote-desktop.path = "../iron-remote-desktop"
 # WASM
 wasm-bindgen = "0.2"
 wasm-bindgen-futures = "0.4"
+wasm-bindgen-derive = "0.3"
```

Comment:
I have tried to use `wasm-bindgen` `convert` module to be able to downcast from ` JsValue` to an extension, but I have found that this module is unstable and gives different results each call (first call - fine, everything else - errors). This is a known issue: https://github.com/rustwasm/wasm-bindgen/issues/2231.
This crate fixes the issue.